src/main/java/org/broadinstitute/hellbender/utils/read/GATKRead.java

```diff
@@ -0,0 +1,255 @@
+     */
+    Cigar getCigar();
+
+    /**
+     * @return The ID of the read group this read belongs to, or {@code null} for none.
+     */
+    String getReadGroup();
+
+    /**
+     * @return Total number of reads in this read's fragment, or 0 if unknown.
+     */
+    int getNumberOfReadsInFragment();
+
+    /**
+     * @return The read number in sequencing; 1 for first of pair, 2 for second of pair.
+     *         1-based and <= {@link #getNumberOfReadsInFragment}
+     */
+    int getReadNumber();
+
+    /**
+     * @return True if this read is paired (ie., {@link #getNumberOfReadsInFragment} == 2), otherwise false.
+     * @throws org.broadinstitute.hellbender.exceptions.GATKException.MissingReadField if this information is not available
+     */
+    boolean isPaired();
+
+    /**
+     * @return True if this read is paired and the orientation and the distance between reads from the fragment are
+     *         consistent with the sequencing protocol, otherwise false.
+     * @throws org.broadinstitute.hellbender.exceptions.GATKException.MissingReadField if this information is not available
+     */
+    boolean isProperlyPaired();
+
+    /**
+     * @return True if this read is unmapped (this includes reads that have a position but are explicitly marked as unmapped,
+     *         as well as reads that lack a fully-defined position but are not explicitly marked as unmapped). Otherwise false.
+     */
+    boolean isUnmapped();
+
+    /**
+     * @return True if this read's mate is unmapped (this includes mates that have a position but are explicitly marked as unmapped,
+     *         as well as mates that lack a fully-defined position but are not explicitly marked as unmapped). Otherwise false.
+     */
+    boolean mateIsUnmapped();
+
+    /**
+     * @return True if this read is on the reverse strand as opposed to the forward strand, otherwise false.
+     * @throws org.broadinstitute.hellbender.exceptions.GATKException.MissingReadField if this information is not available
+     */
+    boolean isReverseStrand();
+
+    /**
+     * @return True if this read's mate is on the reverse strand as opposed to the forward strand, otherwise false.
+     * @throws org.broadinstitute.hellbender.exceptions.GATKException.MissingReadField if this information is not available
+     */
+    boolean mateIsReverseStrand();
+
+    /**
+     * @return True if this read is paired and is the first read in the pair, otherwise false.
+     * @throws org.broadinstitute.hellbender.exceptions.GATKException.MissingReadField if this information is not available
+     */
+    boolean isFirstOfPair();
+
+    /**
+     * @return True if this read is paired and is the second read in the pair, otherwise false.
+     * @throws org.broadinstitute.hellbender.exceptions.GATKException.MissingReadField if this information is not available
+     */
+    boolean isSecondOfPair();
```

Comment:
Ultimately these should be replaced with a GA4GH-style `getReadNumber` method, but we are not ready for this yet. Will have to be addressed in a future ticket.